tasks/shell_crmsh/crm-cib-constraint-colocation.yml

```
@@ -1,11 +1,33 @@
 # SPDX-License-Identifier: MIT
 ---
+# Verify if constraint.resource_leader.id exists
+- name: Verify resource_leader presence {{ constraint.resource_leader.id }}
+  ansible.builtin.command:
+    cmd: |
```

```suggestion
    cmd: >-
```
?  That is, should the command be one line `crm -c {{ __ha_cluster_crm_shadow }} configure show {{ constraint.resource_leader.id }}` ?  because the yaml string `|` operator will preserve newlines, so the command will have newlines in it

---

@richm I have run it with pipes too many times and it never resulted in error, crmsh was able to handle multi line input.

---

@richm  I have changed all pipes where it was necessary, change is in latest commit.